src/components/AssetDetailsPanel/stories/examples.story.mdx

```diff
@@ -0,0 +1,77 @@
+import { action } from '@storybook/addon-actions';
+import { Meta, Story, Preview } from '@storybook/addon-docs/blocks';
+import { AssetDetailsPanel } from '../AssetDetailsPanel';
+import { decorators,  customElementRendering, handleAssetLoaded, onAssetLoaded, toCategory, customStyle} from './helper';
+
+<Meta title="Assets|AssetDetailsPanel/Example" decorators={decorators} />
+
+## Custom categories
+
+
+```jsx
+const toCategory = (name: string): string => name.split('_')[0];
+```
+
+<Preview>
+  <Story name="Custom categories">
+        <AssetDetailsPanel
+      assetId={4650652196144007}
+      toCategory={toCategory}
+      categoryPriorityList={['SOURCE']}
+    />
+  </Story>
+</Preview>
+
+## Custom Loading Spinner
+
+<Preview>
+  <Story name="Custom Loading Spinner ">
+    <AssetDetailsPanel
+      assetId={4650652196144007}
+      customSpinner={<div>Loading...</div>}
+    />
+  </Story>
+</Preview>
+
+## Custom Styles
+
+```jsx
+import CSS from 'csstype';
```

Comment:
That's wrong.  Check the type of `customStyle` prop in code (`AssetDetailsPanelPure.tsx` file).